manifests/install.pp

```diff
@@ -40,9 +40,9 @@
 
   if !$::vault::disable_mlock {
     exec { "setcap cap_ipc_lock=+ep ${vault_bin}":
```

Comment:
btw, would you like me to do a git rebase -i on the commits I did to get the tests passing or do you prefer the have them individually?